codebits.txt

```diff
@@ -0,0 +1,85 @@
+Issues
```

Comment:
Heh, my bad on that. That was left over from development when I was storing some code in case I needed to revert changes. Deleting...